webapp/pages/terms-and-conditions.vue

```diff
@@ -19,5 +77,13 @@ export default {
       title: this.$t('site.termsAc'),
     }
   },
+  data() {
+    return {
+      confirmed: false,
+      latestDate_TaC: '17.10.2008',
+      latestDate_TaC_update: '23.10.2008',
+      latestDate_TaC_UserConfirmed: '18.10.2008',
```

Comment:
```suggestion
```
This date is individual for every user and needs to be saved in the database. Therefore it cannot be hard coded like this.

By the way: We don't *have* to implement it right now. We can implement it once we change our terms and conditions for the first time.